README.md

```diff
@@ -31,4 +31,19 @@ Then import it into python.
 ## Testing
 
 This package uses [pytest](https://pytest.org/) for tests.
-To run tests locally, execute the command `pytest` from the command line.
+To run tests locally, execute the command `pytest` from the command line:
+
+```
+>>>pytest
+```
+
+### Fake data for testing
```

Comment:
what about a word that is nicer than Fake? `### Data generated for testing`

Reply:
Good idea - made this change!